packages/blockchain-wallet-v4-frontend/src/modals/Swap/EnterAmount/Checkout/index.tsx

```diff
@@ -33,10 +34,24 @@ const Amounts = styled.div`
 const MinMaxButtons = styled.div`
   display: flex;
 `
+const ButtonsRow = styled(MinMaxButtons)`
+  justify-content: space-between;
+  width: 105%;
```

Comment:
red flag, whats up with this?

Reply:
Wanted to increase the width of the container so there would be space in between the buttons. Looked ok on mobile, just adding margin around them didn't work. Bad idea?